docs/content.md

```diff
@@ -442,3 +469,99 @@ When updating not just the text content within the file, but the actual list of
+4. Right hand click and select 'New file'
+5. Give name to file ie. when-to-apply.md **make sure you put md on the end of the name if it is a markdown file**
+
+Now you have created a page - you will need to fill in the top details:
+
+**If you have created a page that will appear in one of the category pages ie. How to apply**
+
+title: 	title will appear in the browser tab
+
+heading: this is the h1 for the page - only needed if you want the h1 and the title to be different. If not, just add the title
+
+description: must start with |- and then you can write the text here
+
+related_content: list your related content links here, formatted as internal links
+
+navigation:  this is to decide where the tile for this page will sit on the page - you will need to check what the other pages are listed as. They will be formatted like '30.20' - '30' will stand for the category page (in this case, how to apply) and the '20' will be the order it sits in on the tiles. These usually are assigned in increments of 5, just in case you later want to move on further up it can be easily moved by assigning an in between number.  
+
+navigation_title: This is the title that will appear in the tiles
+
+navigation_description: This is the description that will appear in the tiles
+
+keywords: use Semrush to put keywords here, do a bulletpointed list using dashes like below:
+  - adviser
+  - advisor
+  - ITT
+
+Here is a blank version to take and fill in:
+title:
+heading:
+description: 
+related_content: 
+navigation:  
+navigation_title: 
+navigation_description: 
+keywords: 
+
+## Preview a change
+1. Go to the terminal tab on Visual Studio Code **or** Go the top bar of your laptop - click terminal - click 'new terminal'
+2. A line of code will come up and you should type in **bin/dev**, this will start a series of code
+3. When it has finished running, go into the ports tab and click the globe icon by 127.0.0.1:3000. This will open the browser
+4. When putting in a page to preview it, right click the page title on the left hand side bar in Visual Studio Code and copy path
+5. Go to the url: http://127.0.0.1:3000 and insert a / on the end
+6. The copied path will be something like this: /workspaces/get-into-teaching-app/app/views/content/a-day-in-the-life-of-a-teacher.md
+7. Edit this path to only be the necessary page path: /a-day-in-the-life-of-a-teacher
+
+The url is http://127.0.0.1:3000/a-day-in-the-life-of-a-teacher
+
+## Saving a change
+When you have made a change in Visual Studio Code and want to save your work.
+1. Save the change by CTRL-S
+2. This will cause a notification on the source control
+3. You will need to type a name for this change in the 'message' box
+4. Click the plus button that appears on the change - this will stage the change
+5. Click 'sync changes'
+6. Commit the changes
+7. This will then open a pull request, you will need to fill in the details for this and click 'Create pull request'
```

Comment:
@caitlin-reidy does it automatically create a PR if you commit changes or do you need to click create pull request? I thought you could work locally on pages without creating a PR? But I may have totally misunderstood and been creating PRs without realising!

Reply:
I've added this to the backlog because i have no idea!